src/stores/BreadcrumbsStore.ts

```diff
@@ -51,7 +52,9 @@ export class BreadcrumbsStore extends AsyncStoreWithClient<IState> {
     }
 
     public get visible(): boolean {
-        return this.state.enabled;
+        // @ts-ignore - TypeScript really wants this to be [tagId: string] but we know better.
+        const roomCount = _reduce(RoomListStoreTempProxy.getRoomLists(), (result, rooms) => result + rooms.length, 0);
+        return roomCount >= 20;
```

Comment:
To disable the breadcrumbs? Or to force the display of breadcrumbs?

Reply:
Thanks for the tip

Reply:
We still need to support `this.state.enabled` because the user can turn them off without us. Looks like the previous `RoomBreadcrumbs` watched for membership events then did a `!this.state.enabled && joinedRooms >= NUM` to turn them on.